docs/guide/custom-widgets.md

```diff
@@ -72,6 +72,69 @@ require('apostrophe')({
 });
 ```
 
+### Adding placeholder content
+
+Much like the core widgets, you can add placeholder content for many of the fields in your custom widgets. Within the widget options add `placeholder: true`. Then within individual fields add a `placeholder` property with a value appropriate to the type of schema field.
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  extend: '@apostrophecms/widget-type',
+  options: {
+    label: 'Custom Widget',
+    icon: 'text-subject',
+    placeholder: true,
+    placeholderClass: 'my-custom-widget-placeholder'
+  },
+  fields: {
+    add: {
+      email: {
+        type: 'email',
+        placeholder: 'bob@email.com'
+      },
+      float: {
+        type: 'float',
+        placeholder: 0.05
+      },
+      integer: {
+        type: 'integer',
+        placeholder: 2
+      },
+      string: {
+        type: 'string',
+        placeholder: 'widget-with-placeholder placeholder'
+      },
+      url: {
+        type: 'url',
+        placeholder: 'https://example.com'
+      }
+    }
+  }
+};
+
+```
+
+<template v-slot:caption>
+/modules/my-custom-widget/index.js
+</template>
+</AposCodeBlock>
+
+This placeholder content will be present on the page when the widget is added, but won't appear in the page preview until it is edited. This data will also show up in the fields within the editor module. The `placeholder` option can also be used to fill the editor modal fields with suggested content without also adding it to the page by setting it to `false`, but still adding placeholder content to each field.
```

Comment:
The placeholder content will *never* appear in the page preview. It will *only* appear on-page until you click Edit for that widget and save some real content.

Reply:
editor module -> editor modal

Reply:
Added and changed.